db/hybrid_logical_vector_test.go

```diff
@@ -229,3 +229,56 @@ func TestHybridLogicalVectorPersistence(t *testing.T) {
 	assert.Equal(t, inMemoryHLV.PreviousVersions, hlvFromPersistance.PreviousVersions)
 	assert.Equal(t, inMemoryHLV.MergeVersions, hlvFromPersistance.MergeVersions)
 }
+
+func TestAddNewerVerionsBetweenTwoVectors(t *testing.T) {
```

Comment:
This seems like it would be an excellent candidate for subtests.  Defining a struct with 'localHLV', 'incomingHLV' and 'expectedHLV' as properties would make these tests very easy to quickly read and verify they are testing the right thing.  e.g.
```
{
 localHLV: []string{"abc@15"},
 incomingHLV: []string{"def@25", "abc@20"},
 expectedHLV: []string{"def@25", "abc@20"},
}, ...
```